sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/SdkVersionUtils.cs

```diff
@@ -32,25 +32,17 @@ private static string GetSdkVersion()
 
         private static string GetVersion(Type type)
         {
-            try
-            {
-                string versionString = type
-                    .Assembly
-                    .GetCustomAttributes<AssemblyInformationalVersionAttribute>()
-                    .First()
-                    .InformationalVersion;
+            string versionString = type
+                .Assembly
+                .GetCustomAttributes<AssemblyInformationalVersionAttribute>()
```

Comment:
Is there confidence that this line wouldn't throw exception?

Reply:
This line probably won't. if the attribute is not present it will return empty collection. it will fail on First(). The exception should get caught in the try{}catch() we have in GetSdkVersion()
From docs:
//
        // Summary:
        //     Retrieves a collection of custom attributes of a specified type that are applied
        //     to a specified assembly.
        //
        // Parameters:
        //   element:
        //     The assembly to inspect.
        //
        // Type parameters:
        //   T:
        //     The type of attribute to search for.
        //
        // Returns:
        //     A collection of the custom attributes that are applied to element and that match
        //     T, or an empty collection if no such attributes exist.
        //
        // Exceptions:
        //   T:System.ArgumentNullException:
        //     element is null.